src/Serilog.Sinks.TextWriter/TextWriterLoggerConfigurationExtensions.cs

```diff
@@ -57,5 +58,30 @@ public static class TextWriterLoggerConfigurationExtensions
             var sink = new TextWriterSink(textWriter, formatter);
             return sinkConfiguration.Sink(sink, restrictedToMinimumLevel, levelSwitch);
         }
+
+        /// <summary>
+        /// Write log events to the provided <see cref="System.IO.TextWriter"/>.
+        /// </summary>
+        /// <param name="sinkConfiguration">Logger sink configuration.</param>
+        /// <param name="textWriter">The text writer to write log events to.</param>
+        /// <param name="textFormatter">Text formatter used by sink.</param>
+        /// /// <param name="restrictedToMinimumLevel">The minimum level for
+        /// events passed through the sink. Ignored when <paramref name="levelSwitch"/> is specified.</param>
+        /// <param name="levelSwitch">A switch allowing the pass-through minimum level
+        /// to be changed at runtime.</param>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static LoggerConfiguration TextWriter(
+            this LoggerSinkConfiguration sinkConfiguration,
+            TextWriter textWriter,
+            ITextFormatter textFormatter,
```

Comment:
By convention, in other sinks that provide this overload the formatter argument is first (after 'this') and called `formatter` (e.g. File, RollingFile)